nette/notejam/app/components/Notes/Notes.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Components\Notes;
+
+use Nette;
+use Nette\Database\Table\Selection;
+
+
+class Notes extends Nette\Application\UI\Control
+{
+
+	/** @var Selection */
+	private $notes;
+
+	/** @var object */
+	private $pad;
+
+	/**
+	 * Pads constructor.
+	 * @param Selection $notes
+	 * @param object    $pad
+	 */
+	public function __construct($notes, $pad = null)
+	{
+		$this->notes = $notes;
+		$this->pad = $pad;
+	}
+
+
+	/**
+	 * Renders the component.
+	 */
+	public function render()
+	{
+		$template = $this->createTemplate();
+		$template->setFile(__DIR__ . '/Notes.latte');
```

Comment:
Huh, I didn't know you could do this since the `Nette\Application\UI\ITemplate::render()` doesn't support any parameters :/